content/json-ld-crud-api/latest/events/event-bookingavailability.md

```diff
@@ -0,0 +1,94 @@
+---
+---
+
+# Event - bookingAvailability
+
+The bookingAvailability indicates whether there are still tickets or seats available for an event The following values for 'bookingAvailability' are applicable:
+* `available`: there are still tickets or seats left for the event, or the capacity for the event is unlimited.
+* `unavailable`: the event is sold out or fully booked.
```

Comment:
Should be `Available` and `Unavailable` (capitalized). Might also need to be fixed in other places.

Reply:
Thanks 👍 . Fixed it